doc/source/cli_admin_examples.rst

```diff
@@ -0,0 +1,127 @@
+========================
+Rucio administration CLI
+========================
+
+
+Rucio provides a CLI for administrative tasks.. The get methods can be executed by any users, but the set methods require some admin privileges::
+
+  $ rucio-admin 
+  usage: rucio-admin [-h] [--version] [--verbose] [-H ADDRESS]
+                     [--auth_host ADDRESS] [-a ACCOUNT] [-S AUTH_STRATEGY]
+                     [-T TIMEOUT] [-u USERNAME] [-pwd PASSWORD]
+                     [--certificate CERTIFICATE]
+                     [--ca-certificate CA_CERTIFICATE]
+                     {account,identity,rse,scope,config,subscription,replicas}
+                     ...
+  
+  positional arguments:
+    {account,identity,rse,scope,config,subscription,replicas}
+      account             Account methods
+      identity            Identity methods
+      rse                 RSE (Rucio Storage Element) methods
+      scope               Scope methods
+      config              Configuration methods
+      subscription        Subscription methods
+      replicas            Replica methods
+  
+  optional arguments:
+    -h, --help            show this help message and exit
+    --version             show program's version number and exit
+    --verbose, -v         Print more verbose output
+    -H ADDRESS, --host ADDRESS
+                          The Rucio API host
+    --auth_host ADDRESS   The Rucio Authentication host
+    -a ACCOUNT, --account ACCOUNT
+                          Rucio account to use
+    -S AUTH_STRATEGY, --auth-strategy AUTH_STRATEGY
+                          Authentication strategy (userpass, x509, ssh ...)
+    -T TIMEOUT, --timeout TIMEOUT
+                          Set all timeout values to SECONDS
+    -u USERNAME, --user USERNAME
+                          username
+    -pwd PASSWORD, --password PASSWORD
+                          password
+    --certificate CERTIFICATE
+                          Client certificate file
+    --ca-certificate CA_CERTIFICATE
+                          CA certificate to verify peer against (SSL)
+
```

Comment:
this part can be generated in a distinct section (see #627)

Reply:
OK. Maybe in the meantime we approve this PR, then I will resubmit one with improved doc in the bin

Reply:
👍